README.md

```diff
@@ -30,6 +30,24 @@ Convert markdown file to HTML and **output content to an HTML file** using `--ou
 $ formatter README.md --output path/to/output/file.html
 ```
 
+Convert markdown file to HTML and **set the language of the HTML document** using `--lang`:
```

Comment:
Would be nice to start adding sub headings: 
```markdown
#### Options

##### Document language (`--lang`)

...

##### Table of contents levels (`--toc`)

...
```

And since they are optional. It would be good to describe the default. (Default language is 'en'. Default heading levels are 2 to 6?)